package.json

```diff
@@ -107,31 +108,30 @@
     "stylelint": "^13.13.1",
     "stylelint-config-recommended-scss": "^4.3.0",
     "stylelint-scss": "^3.19.0",
+    "ts-patch": "^3.1.2",
+    "typescript": "^5.3.3",
     "webpack": "^5.44.0",
     "webpack-bundle-analyzer": "^4.4.2",
     "webpack-cli": "^4.7.2"
   },
   "scripts": {
-    "build": "webpack --config config/prod.webpack.config.js",
-    "build:prod": "NODE_ENV=production webpack --config config/prod.webpack.config.js",
+    "build": "fec build",
     "test:jest": "jest --env=jsdom",
     "test": "TZ=UTC jest --verbose --no-cache",
     "lint": "npm-run-all lint:*",
     "lint-fix": "eslint config src --fix",
     "lint:js": "eslint config src",
     "lint:sass": "stylelint 'src/**/*.scss' --config .stylelintrc.json",
     "server:ctr": "node src/server/generateServerKey.js",
-    "start": "NODE_ENV=development webpack serve --config config/dev.webpack.config.js",
-    "start:beta": "NODE_ENV=development BETA=true webpack serve --config config/dev.webpack.config.js",
-    "start:proxy": "PROXY=true npm run start",
-    "start:proxy:beta": "BETA=true npm run start:proxy",
-    "travis:build": "NODE_ENV=production webpack --config config/test.webpack.config.js",
-    "travis:verify": "npm-run-all build:prod lint test:ct",
+    "start": "PROXY=true HOT=true fec dev",
+    "start:linking": "PROXY=false port=8003 HOT=true fec dev",
```

Comment:
Other apps use just `npm run start` to only start the web pack devserver without proxy functionality and `npm run start:proxy` to start it with proxy functionality. We should keep it that way.

Reply:
We could make hot reloading default to true in the config and remove it from here.
If someone would want to disable hot reloading they would need to edit the commands or config, because running `HOT=false npm run start` would have no effect, because the `HOT` in the run script will take precedence.

Reply:
Yeah, you are right, especially regarding the consistency with other apps and guidelines. I thought it would be better to make the command shorter. We do not always run the dev server without a proxy. The only case when we use it is when we link the app together and I created a separate readily-available command for it.

Reply:
Will push these changes.

Reply:
Using the `npm run start` to only start the web pack dev server is to be consistent with other/general React/Webpack projects, which makes it easier for others to approach our projects, when they are not familiar with them yet.

Reply:
@bastilian done.